### PR TITLE
[teleop] Add conversions for chlorophyll spectral

### PR DESCRIPTION
**Pull request by CameronTressler**

### CODE REVIEW

**Comment by CameronTressler** [comment]
Message from Gabe:

Here are the numbers for the conversion from spectral 2 and spectral 1 to spectral 0. It's kind of hard to label these clearly but "x1" and "x2" are the measured values currently being printed out for spectral sensors 1 and 2, and "y" is the converted value for each.

Spectral 1: y1 = (1 / 0.94) * (x1 - 23.28)
Spectral 2: y2 = (1 / 0.9969) * (x2 + 32.8575)

File: base_station/gui/src/components/Chlorophyll.vue [hunk 1]
```diff
@@ -114,6 +114,14 @@ export default {
         'device': id,
         'enable': enabled
       })
+    },
+
+    spectral_1_conversion: function(sensor_reading) {
+      return (1 / 0.94) / (sensor_reading - 23.28)
```
Comment (tabiosg) [inline base_station/gui/src/components/Chlorophyll.vue:120]: Will the gui break if we get a divide by 0 problem

**Review by tabiosg — COMMENTED** [review]
I'll test this later and then I'll approve if it works

File: base_station/gui/src/components/Chlorophyll.vue (hunk 1 above)
Comment (tabiosg) [reply base_station/gui/src/components/Chlorophyll.vue:120]: Actually, Gabe said it was multiply, not divide

**Review by tabiosg — CHANGES_REQUESTED** [review]
Change to multiply not divide in equation i think

**Review by cgiger00 — APPROVED** [review]

**Review by tabiosg — APPROVED** [review]

**Comment by cgiger00** [comment]
NOTE: ***AFTER THE CODE FREEZE*** needed addition for science that we had missed

**Pull request merged**